src/evict/evict_lru.c

```diff
@@ -1538,6 +1538,12 @@ __evict_walk(WT_SESSION_IMPL *session, WT_EVICT_QUEUE *queue)
     if (dhandle_locked)
         __wt_readunlock(session, &conn->dhandle_lock);
 
+    if (incr) {
+        WT_ASSERT(session, dhandle->session_inuse > 0);
+        (void)__wt_atomic_subi32(&dhandle->session_inuse, 1);
+        incr = false;
```

Comment:
This value won't get read again so we should remove this.

Reply:
Hi @tetsuo-cpp 
Ok I will remove this `incr = false` line.
I can also rebase my commit onto the `develop` branch so this PR will contain only single commit. Is this ok?

Reply:
Thanks, @igorsol! No need to rebase since I can squash and merge via the Github UI.